### ISSUE TITLE
Homepage testimonial cards

### ISSUE THREAD

**Issue by autumnfound** [jakartaee/jakarta.ee#1138]
New testimonial blocks will be created that display the logo of the company giving a testimonial on the left and testimonial, name, and company on the right. The name and company will be wrapped in a paragraph tag with the brand-secondary class (or whichever gives the blue color), and the company in the same tag with an em tag wrapping the company name.

The determination of how we choose which testimonials are viewed is TBD. Likely solutions are a highlighted param in the testimonial metadata or some JS that populates the section on load and uses BS fade in to hide the pop in. 

@karenmcnaughton our current testimonials are long and would not fit well in this design. We would need to cut all of them to less than half, or a third of their length for the majority of our entries (140 characters hard max including spaces). We may also find some of the more wide logos to be very cramped in the space we gave them. While this isn't a blocker, it's just a warning that some content may be less than ideal in this space.

Additionally, do we want to randomize the available testimonials on load, or have highlighted testimonials? We have more than the 3 shown, so either some may not be immediately visible, or would be on a sub-page.

Design: 
![image](https://user-images.githubusercontent.com/23018241/117336877-aef52580-ae6a-11eb-8b02-62bc783e4de3.png)

**Comment by autumnfound** [comment]
RE the testimonials to display, we should randomize with weight (weights to be determined by marketing). This will be JS, and require some time to fiddle with.

RE the length, we can chop off the content (no HTML so easy-peasy), and link to a larger page with just testimonials. We don't currently have a mockup for this, but we could do a page with horizontal cards and see how we feel.

**Comment by autumnfound** [comment]
The random weighting portion of this will be moved to a separate issue for post-launch as it's a new feature that currently doesn't exist on the site (AFAIK). This backlog will be completed once the critical redesign work is done, and a few features may make it in to launch if things go smoothly with everything else.

**Comment by autumnfound** [comment]
I had a discussion with @shabnammayel this morning and the following items were discussed:

- request to add a feature of expanding the testimonial on click to show more text to allow longer testimonials
- what length of the testimonial should be targeted. 300 characters was the final number being put forward to the marketing committee later today
- question on how testimonials will link to the main testimonials page. No answer was given here as we haven't implemented anything yet, and no implementation notes exist yet
- question on how they will be displayed and the final decision on strategy. Random 3 on load was suggested and split into another issue (see #1157) with no feedback given from the marketing/design teams.

**Comment by ericpoirier** [comment]
The testimonial cards on the home page has been merged with the site's redesign.

Thanks!
